sogs/__main__.py

```diff
@@ -218,6 +218,7 @@ def print_room(room: Room):
         "+" if room.default_accessible else "-",
     )
 
+    nl = "\n                    "
```

Comment:
if you set nl to something like that it should look something like this instead: `nl = '\n' + (' ' * N)` where N is the number of spaces to pad

Reply:
Good idea. That's much more readable.

Rebased.